src/synthesizer/abundances/elements.py

```diff
@@ -0,0 +1,137 @@
+
+class Elements:
+
+    """
+    A simple class containing various useful lists and dictionaries.
+
+    Attributes:
+        non_metals (list, string)
+            A list of elements classified as non-metals.
+        metals (list, string)
+            A list of elements classified as metals.
+        all_elements (list, string)
+            A list of all elements, functionally the concatenation of metals
+            and non-metals.
+        alpha_elements (list, string)
+            A list of the elements classified as alpha-elements.
+        name (dict, string)
+            A dictionary holding the full name of each element.
+        A (dict, float)
+            Atomic mass of each element (in amus).
+    """
+
+    non_metals = [
+        "H",
+        "He",
+    ]
+
+    metals = [
+        "Li",
+        "Be",
+        "B",
+        "C",
+        "N",
+        "O",
+        "F",
+        "Ne",
+        "Na",
+        "Mg",
+        "Al",
+        "Si",
+        "P",
+        "S",
+        "Cl",
+        "Ar",
+        "K",
+        "Ca",
+        "Sc",
+        "Ti",
+        "V",
+        "Cr",
+        "Mn",
+        "Fe",
+        "Co",
+        "Ni",
+        "Cu",
+        "Zn",
+    ]
+
+    all_elements = non_metals + metals
+
+    # the alpha process elements
+    alpha_elements = [
+        "O",
+        "Ne",
+        "Mg",
+        "Si",
+        "S",
+        "Ar",
+        "Ca",
+        "Ti",
+    ]
+
+    # Name
+    name = {}
+    name["H"] = "Hydrogen"
+    name["He"] = "Helium"
+    name["Li"] = "Lithium"
+    name["Be"] = "Beryllium"
+    name["B"] = "Boron"
+    name["C"] = "Carbon"
+    name["N"] = "Nitrogen"
+    name["O"] = "Oxygen"
+    name["F"] = "Fluorine"
+    name["Ne"] = "Neon"
+    name["Na"] = "Sodium"
+    name["Mg"] = "Magnesium"
+    name["Al"] = "Aluminium"
+    name["Si"] = "Silicon"
+    name["P"] = "Phosphorus"
+    name["S"] = "Sulphur"
+    name["Cl"] = "Chlorine"
+    name["Ar"] = "Argon"
+    name["K"] = "Potassium"
+    name["Ca"] = "Calcium"
+    name["Sc"] = "Scandium"
+    name["Ti"] = "Titanium"
+    name["V"] = "Vanadium"
+    name["Cr"] = "Chromium"
+    name["Mn"] = "Manganese"
+    name["Fe"] = "Iron"
+    name["Co"] = "Cobalt"
+    name["Ni"] = "Nickel"
+    name["Cu"] = "Copper"
+    name["Zn"] = "Zinc"
+
+    # mass of elements in amus
+    A = {}
```

Comment:
should use a pep8 friendly variable name here, not just `A`

Reply:
Done

Reply:
This thing could be a dictionary of dictionaries, or even just a yaml file, but then you've got to worry about the path to it.